index.html

```diff
@@ -159,7 +159,7 @@ <h3>Japanese Text Support</h3>
           <tr>
             <td><code>#textEmphasis-minimal</code></td>
```

Comment:
Is there now a more expressive TTML2 feature designator we can use in place of this?

Reply:
@nigelmegitt What do you mean by "more expressive"?

Reply:
I mean one that more accurately maps to support for/use of the `outside` value.

Reply:
None of the emphasis features relate specifically to the `<emphasis-position>` component of `tts:textEmphasis`

Reply:
Oh, I see that the `#textEmphasis-no-XXX` features are negative features rather than positive ones. I didn't notice that before. I think it's broken in TTML2 and will raise an issue for this: features should be positively stated not negatively, otherwise the future introduction of additional syntax and semantics will change the meaning of existing `-no-XXX` style feature designators.